src/rules/checkLineAlignment.js

```diff
@@ -229,7 +229,7 @@ export default iterateJsdoc(({
     disableWrapIndent = false,
     preserveMainDescriptionPostDelimiter,
     tags: applicableTags = [
-      'param', 'arg', 'argument', 'property', 'prop', 'returns', 'return',
+      'param', 'arg', 'argument', 'property', 'prop', 'returns', 'return', 'template',
```

Comment:
Thanks... Could you add this to the docs, too?

Reply:
Done!